terraform/modules/swipe-sfn-batch-queue/container_instance_user_data

```diff
@@ -27,4 +27,17 @@ mount /dev/md0 MINIWDL_DIR
 
 cloud-init-per once docker_options echo '{"data-root": "MINIWDL_DIR/docker"}' >> /etc/docker/daemon.json
 
+##############################################
+# Set up a job to terminate the instance if it becomes full
+##############################################
+
+echo '#!/bin/bash' > /bin/space-check
+echo '[[ $(df /mnt --output=avail |tail -n1) -lt 1000000 ]] && poweroff' >> /bin/space-check
```

Comment:
We should probably reference `/mnt` as `MINIWDL_DIR` here. 

Can we also have a log here so if this happens we're not so confused as to why the instance shut down? I was thinking something like ```&& poweroff; echo system terminated due to full drive  >> /dev/stderr ```